change/@fluentui-react-charting-779871c9-4df7-4cc5-96b6-832621cec379.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33477&commitId=410c0205b8c887310d62919749b9ae7024213a72&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
VR screenshots unrelated to the change